packages/api/src/controllers/organizations/v2/mod.rs

```diff
@@ -10,39 +9,46 @@ use by_axum::{
     },
 };
 use by_types::DatabaseConfig;
-use deliberations::DeliberationController;
 use models::*;
 use reqwest::StatusCode;
 use sqlx::postgres::PgPoolOptions;
 
+#[derive(
+    Debug, Clone, serde::Deserialize, serde::Serialize, schemars::JsonSchema, aide::OperationIo,
+)]
+pub struct OrganizationPath {
+    pub org_id: i64,
+}
+
 #[derive(Clone, Debug)]
 pub struct OrganizationControllerV2 {}
 
 impl OrganizationControllerV2 {
     pub fn route(pool: sqlx::Pool<sqlx::Postgres>) -> Result<by_axum::axum::Router> {
         Ok(by_axum::axum::Router::new()
             .nest(
-                "/:org-id/deliberations",
-                DeliberationController::new(pool.clone()).route()?,
+                "/:org_id/deliberations",
+                crate::controllers::deliberations::v2::DeliberationController::new(pool.clone())
+                    .route()?,
             )
             .nest(
-                "/:org-id/surveys",
+                "/:org_id/surveys",
                 crate::controllers::survey::v2::SurveyControllerV2::route(pool.clone())?,
             )
             .nest(
-                "/:org-id/panels",
+                "/:org_id/panels",
                 crate::controllers::panels::v2::PanelControllerV2::route(pool.clone())?,
             )
             .nest(
-                "/:org-id/resources",
+                "/:org_id/resources",
                 crate::controllers::resources::v1::ResourceControllerV1::route(pool.clone())?,
             )
             .nest(
-                "/:org-id/members",
+                "/:org_id/members",
```

Comment:
```suggestion
                "/:org-id/members",
```
never change this. it will cause not found.check it out https://github.com/biyard/voice-korea/blob/main/packages/api/src/controllers/v2/organizations/_id/members.rs#L246

and it has been already tested. why did you change all paths even if it does not  be related to your work?

Reply:
I've confirmed that it recognizes properly, but I'll roll back for now.